Automattic/woocommerce-subscriptions-core
# Use admin theme color

Fixes #490 

## Description
There are a few places that are using some hard-coded pink colors across the extension. And some selectors are not following the admin theme color. Changed the colors here to make it consistent with WooCommerce and the admin theme.

Changes made in this PR are-
- Changed the color of the info icon on `Payments` page to follow the admin theme color.
- Removed unnecessary button styles. With them removed, the WooCommerce button colors would be picked up following the admin theme color.
- Replaced the pink colors (`#9c5d90`, `#ddc8d9`, `#cc99c2`) with WooCommerce purple color (`#7F54B3`)
- Added `wc-enhanced-select` class to the selectors which makes them follow the admin theme color.

## Testing instructions
- Ensure the codebase does not have any wrong pink colors left.
- Go to `<your_site/wp-admin/profile.php` and choose an Admin Color Scheme except for `default`. (I have used `Modern` in the screenshots)
- Go to **WooCommerce > Settings > Payments** page. The info icon should follow the admin theme color.

**Before**

<img width="50%" height="50%" src="https://github.com/Automattic/woocommerce-subscriptions-core/assets/33387139/8c2bf55b-97c2-4087-be56-8d9a96ca1076" />

**After**

<img width="50%" height="50%" src="https://github.com/Automattic/woocommerce-subscriptions-core/assets/33387139/56aec9df-f46f-417d-b76e-d4c7710eb8c6" />

- Go to **WooCommerce > Emails** and go inside the pages `Cancelled Subscription`, `Expired Subscription`, `Suspended Subscription`. The `Email Type` selector should follow the admin theme color.


<img width="50%" height="50%" src="https://github.com/Automattic/woocommerce-subscriptions-core/assets/33387139/5ac9e88b-ee3f-4a74-9467-c32dd110024f" />

- Go to **WooCommerce > Product > Add New**. Select `Simple Subscription` and check the `general` tab. The selectors related to subscription should follow the admin theme color.


<img width="50%" height="50%" src="https://github.com/Automattic/woocommerce-subscriptions-core/assets/33387139/d2d10897-3cc3-4f40-89fa-a9ab5dfd333c" />


- Go to **WooCommerce > Product > Add New**. Select `Variable Subscription` and check the `variation` tab. The selectors related to subscription should follow the admin theme color.


<img width="50%" height="50%" src="https://github.com/Automattic/woocommerce-subscriptions-core/assets/33387139/8bea2bb2-705c-430d-931e-8791e5a9f55a" />
